objects/src/notes/mod.rs

```diff
@@ -79,6 +79,11 @@ impl Note {
         self.serial_num
     }
 
+    /// Returns the note data as a vector of elements.
+    pub fn as_elements(&self) -> Vec<Felt> {
+        self.into()
+    }
+
```

Comment:
minor nit: I'd probably rename this to `to_elements()`. I usually use `as_` when the function does not result in allocations (i.e., returns a slice).

Reply:
Looks like this is also recommended by the [rust API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv).